scripts/storage-emulator-integration/tests.ts

```diff
@@ -1260,19 +1243,23 @@ describe("Storage emulator", () => {
       await testBucket.upload(image_filename, { destination: filename });
     });
 
-    afterEach(async () => {
+    afterEach(async function (this) {
+      this.timeout(EMULATORS_SHUTDOWN_DELAY_MS);
+      await page.evaluate(async () => {
+        await firebase.auth().signOut();
+      });
       await page.close();
     });
 
-    after(async () => {
+    after(async function (this) {
+      this.timeout(EMULATORS_SHUTDOWN_DELAY_MS);
```

Comment:
We set this in line 104, I think we can remove this. Unless we want the same timeout applied to Firebase Endpoints and the entire test suite?

Reply:
Done